mods/ra/maps/soviet-04b/main.lua

```diff
@@ -0,0 +1,150 @@
+CheckForRadar = function()
+	if not Radar.IsDead then
+		return true
+	end
+end
+
+CheckForCY = function()
+	if not CYard.IsDead then
+		return true
+	end
+end
+
+RunInitialActivities = function()
+	Harvester.FindResources()
+	Helper.Destroy()
+	IdlingUnits()
+	Trigger.AfterDelay(DateTime.Seconds(1), function()
+		BringPatrol1()
+		Trigger.AfterDelay(DateTime.Seconds(5), function()
+			BringPatrol2()
+		end)
+		BuildBase()
+
+		local buildings = Map.ActorsInBox(Map.TopLeft, Map.BottomRight, function(self) return self.Owner == Greece and self.HasProperty("StartBuildingRepairs") end)
```

Comment:
Try to avoid https://github.com/OpenRA/OpenRA/issues/8549 everywhere. Use `Map.NamedActors` instead. See https://github.com/OpenRA/OpenRA/blob/bleed/mods/cnc/maps/gdi03/gdi03.lua#L79